libs/platform/src/lib/components/form/form-group/form-field-group/form-field-group.component.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ContentChildren,
+    Input,
+    OnInit,
+    QueryList,
+    ViewEncapsulation
+} from '@angular/core';
+import { FormField } from '../../form-field';
+import { FormGroupContainer} from '../../form-group';
+import { FormFieldGroup } from '../../form-field-group';
+
+@Component({
+    selector: 'fdp-form-field-group',
+    template: `<ng-content></ng-content>`,
+    styleUrls: ['./form-field-group.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None
+})
+export class FormFieldGroupComponent implements FormFieldGroup, OnInit {
+    /** Group header title*/
+    @Input()
+    label: string;
+
+    /** Get form fields wrapped into form field group */
+    @ContentChildren(FormField) fields: QueryList<FormField>;
+
+    constructor(
+        private readonly formGroupContainer: FormGroupContainer
+    ) {
+    }
+
+    /** @hidden */
+    ngOnInit(): void {
+        this._addFormFieldGroup();
+    }
+
+    /** @hidden */
+    private _addFormFieldGroup(): void {
+        this.formGroupContainer.addFormField(this);
```

Comment:
`FormFieldGroup` just a wrapper of `FormField`s with specified properties. We could check grouping fields in `FormField`, but it would not be easy to grouping fields into a group

Reply:
let's do that by separate method